installer/install.iss

```diff
@@ -989,7 +997,10 @@ begin
 #ifdef GIT_VERSION
     if Result or (CountDots(CurrentVersion)>3) or (CountDots(PreviousGitForWindowsVersion)>3) then begin
         // maybe the previous version was a prerelease (prereleases have five numbers: v2.24.0-rc1.windows.1 reduces to '2.24.0.1.1')?
-        CurrentVersion:='{#GIT_VERSION}';
+        if CurrentVersion=ExpandConstant('{#APP_VERSION}') then
+            CurrentVersion:='{#GIT_VERSION}'
+        else
+            CurrentVersion:='git version '+CurrentVersion+'.windows.1';
```

Comment:
I'm probably misreading this, but wouldn't the setting `CurrentVersion` to `GIT_VERSION` yield something like "v2.33.0-rc2.windows.1", but setting it to that custom string would get something like `git version v2.33.0-rc2.windows.1`? Why is the `git version` prefix needed?

Reply:
`GIT_VERSION` is set here: https://github.com/git-for-windows/build-extra/blob/37805f958614c014fb86bbae91f08d11f9f90821/installer/release.sh#L165-L166

As you can see, it is the output of `git version`, unprocessed. The `git version ` prefix is part of that output.